app/grants/clr.py

```diff
@@ -346,30 +354,40 @@ def fetch_data(clr_type=None, network='mainnet', clr_start_date=None, clr_end_da
 
     if clr_type == 'tech':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='tech', link_to_new_grant=None)
-        threshold = THRESHOLD_TECH
+        v_threshold = V_THRESHOLD_TECH
+        uv_threshold = UV_THRESHOLD_TECH
         total_pot = TOTAL_POT_TECH
+
     elif clr_type == 'media':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='media', link_to_new_grant=None)
-        threshold = THRESHOLD_MEDIA
+        v_threshold = V_THRESHOLD_MEDIA
+        uv_threshold = UV_THRESHOLD_MEDIA
         total_pot = TOTAL_POT_MEDIA
+
     elif clr_type == 'health':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='health', link_to_new_grant=None)
-        threshold = THRESHOLD_HEALTH
+        v_threshold = V_THRESHOLD_HEALTH
+        uv_threshold = UV_THRESHOLD_HEALTH
         total_pot = TOTAL_POT_HEALTH
+
     elif clr_type == 'change':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='change', link_to_new_grant=None)
-        threshold = THRESHOLD_CHANGE
+        v_threshold = V_THRESHOLD_CHANGE
+        uv_threshold = UV_THRESHOLD_CHANGE
         total_pot = TOTAL_POT_CHANGE
+
     elif clr_type == 'matic':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='matic', link_to_new_grant=None)
-        threshold = THRESHOLD_MATIC
+        v_threshold = V_THRESHOLD_MATIC
+        uv_threshold = UV_THRESHOLD_MATIC
         total_pot = TOTAL_POT_MATIC
+
     else:
-        return None, None, None, None
+        return None, None, None, None, None, None
```

Comment:
is the purpose of this to not have it error out if its empty?

Reply:
Yup! Just being safe